microsoft-edge/webview2/release-notes.md

```diff
@@ -1107,3 +1185,68 @@ Initial developer preview release.
+[Webview2ReferenceWin32Icorewebview2experimentalframeViewWebview210999PrereleaseRemoveNavigationcompleted]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe?view=webview2-1.0.999-prerelease&preserve-view=true#remove_navigationcompleted "remove_NavigationCompleted - interface ICoreWebView2ExperimentalFrame | Microsoft Docs"
+[Webview2ReferenceWin32Icorewebview2experimentalframeViewWebview210999PrereleaseAddDomcontentloaded]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe?view=webview2-1.0.999-prerelease&preserve-view=true#add_domcontentloaded "add_DOMContentLoaded - interface ICoreWebView2ExperimentalFrame | Microsoft Docs"
+[Webview2ReferenceWin32Icorewebview2experimentalframeViewWebview210999PrereleaseRemoveDomcontentloaded]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe?view=webview2-1.0.999-prerelease&preserve-view=true#remove_domcontentloaded "remove_DOMContentLoaded - interface ICoreWebView2ExperimentalFrame | Microsoft Docs"
+[Webview2ReferenceWin32Icorewebview2experimentalframeViewWebview210999PrereleaseExecutescript]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe?view=webview2-1.0.999-prerelease&preserve-view=true#executescript "ExecuteScript - interface ICoreWebView2ExperimentalFrame | Microsoft Docs"
+[Webview2ReferenceWin32Icorewebview2experimentalframenavigationstartingeventhandlerViewWebview210999Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalframenavigationstartingeventhandler?view=webview2-1.0.999-prerelease&preserve-view=true "interface ICoreWebView2ExperimentalFrameNavigationStartingEventHandler | Microsoft Docs"
+[Webview2ReferenceWin32Icorewebview2experimentalframecontentloadingeventhandlerViewWebview210999Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalframecontentloadingeventhandler?view=webview2-1.0.999-prerelease&preserve-view=true "interface ICoreWebView2ExperimentalFrameContentLoadingEventHandler | Microsoft Docs"
+[Webview2ReferenceWin32Icorewebview2experimentalframenavigationcompletedeventhandlerViewWebview210999Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalframenavigationcompletedeventhandler?view=webview2-1.0.999-prerelease&preserve-view=true "interface ICoreWebView2ExperimentalFrameNavigationCompletedEventHandler | Microsoft Docs"
+[Webview2ReferenceWin32Icorewebview2experimentalframedomcontentloadedeventhandlerViewWebview210999Prerelease]: /microsoft-edge/webview2/reference/win32/icorewebview2experimentalframedomcontentloadedeventhandler?view=webview2-1.0.999-prerelease&preserve-view=true "interface ICoreWebView2ExperimentalFrameDOMContentLoadedEventHandler | Microsoft Docs"
+
+## To stable
```

Comment:
```suggestion
<!-- To stable -->
```
Are these intended to be visible headings in the rendered page, or just hidden HTML comment divider lines?